Util.java

```diff
@@ -13,12 +13,24 @@
 	 *
 	 * @param var
 	 *        The floating point number you want to compare
+	 * @param epsilon
+	 *		  The maximum value of var such that it can be considered zero.
```

Comment:
I'd probably opt for "magnitude" rather than "value" here. Drop the `.` at the end for consistency. Also, it seems that the indentation is a bit messed up in this file; running the formatter should (hopefully) fix it.